ptbcontrib/persistent_jobstore/requirements.txt

```diff
@@ -0,0 +1,3 @@
+python-telegram-bot>=13.0
+SQLAlchemy
```

Comment:
does this need a pinned/range-restricted version?

Reply:
Not really, works file with older versions as well as latest one